app/services/stripe/invoice_handler.rb

```diff
@@ -0,0 +1,36 @@
+class InvoiceHandler
+  def initialize(event)
+    @invoice = event.data.object
+  end
+
+  def create_invoice!
+    begin
+    	Invoice.create!(
+        brand:              get_brand,
+        stripe_invoice_id:  @invoice.id,
+        amount:             @invoice.amount_due,
+        data:               @invoice,
+        paid_at:            Time.now.utc
```

Comment:
Why are we setting the `paid_at` timestamp here? This just the `created` event not the `payment_succeeded` event, right?